### ISSUE TITLE
It isn't clear who is the maintainer of this plugin

### ISSUE THREAD

**Issue by regisb** [overhangio/tutor-forum#12]
@ghassanmas please implemented the changes recommended in this PR: https://github.com/overhangio/tutor-ecommerce/pull/34

**Comment by ghassanmas** [comment]
Yes, I was thinking of doing this along with olive changes, but I can open a dedicated PR since its not really realted. 
By the way should this bit be updated as well https://github.com/overhangio/tutor-forum/blob/0727396af6fc0c01f7dab53c98e1223acd78058f/tutorforum/templates/forum/build/forum/Dockerfile#L2 
In any case I think it should be changed to `LABEL` [ref](https://github.com/overhangio/tutor-forum/blob/0727396af6fc0c01f7dab53c98e1223acd78058f/tutorforum/templates/forum/build/forum/Dockerfile#L2)

**Comment by regisb** [comment]
> Yes, I was thinking of doing this along with olive changes

Sounds good. Just add `Close #12` to your PR commit message.

> In any case I think it should be changed to LABEL [ref](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)

Are there any standard authors/maintainers label for Docker? I don't think there is:

```
$ docker inspect python:3.8-alpine | grep -i Label
            "Labels": {}
            "Labels": null
$ docker inspect ubuntu:22.04 | grep -i Label
            "Labels": {}
            "Labels": null
```

In which case, we should simply get rid of the deprecated MAINTAINER line. But if I'm wrong we should use the "right" standard labels. What do you think?

**Comment by ghassanmas** [comment]
Yes I agree to remove `MAINTAINER` line. 

For the replacement, yeah I think `LABEL` might not be widely adopted, I think its related to the [Open Container Initiative](https://github.com/opencontainers/image-spec/blob/main/annotations.md) standard which Docker suppose to follow... 

Here I found example of usage of Caddy:

```Dockerfile
LABEL org.opencontainers.image.version=v2.6.2
LABEL org.opencontainers.image.title=Caddy
LABEL org.opencontainers.image.description="a powerful, enterprise-ready, open source web server with automatic HTTPS written in Go"
LABEL org.opencontainers.image.url=https://caddyserver.com
LABEL org.opencontainers.image.documentation=https://caddyserver.com/docs
LABEL org.opencontainers.image.vendor="Light Code Labs"
LABEL org.opencontainers.image.licenses=Apache-2.0
LABEL org.opencontainers.image.source="https://github.com/caddyserver/caddy-docker"
```
Ref: https://github.com/caddyserver/caddy-docker/blob/aca93aecdbd9ba4b4229fba1e6c7ee7b87d5cf7f/2.6/alpine/Dockerfile#L44-L51

So I beleive some of these metadata suppose to already exists in a tutor plugin, and some can be prefilled with the template. 

I don't have strong opinion on one hand, and on another if each tutor Dockerfile is suppose to live/run inside a plugin which has its metadata, it might feel like redundant, _unless its expected that dokcer imag to be used outside of it's plugin context_... i.e. I use tutor-forum image but without installing the plugin...

**Comment by regisb** [comment]
Let's not create LABELs for now. I'll keep your idea in mind if need arises.
